source/_integrations/lcn.markdown

```diff
@@ -66,7 +66,77 @@ The configuration of the `lcn` devices and entities is completely done using a w
+
+![LCN Configuration Panel](/images/integrations/lcn/lcn_device_page.png)
+
+### Configuring devices
+
+You can add and remove modules and groups directly from the configuration panel. Once added, they will appear in Home Assistant as devices, which can be used to trigger [specific actions](https://www.home-assistant.io/integrations/lcn/#actions) within scripts or automations. For examples, refer to the [Performing actions](https://www.home-assistant.io/docs/scripts/perform-actions/) page.
+
+The **Modules / Groups** tab provides an overview of your configured LCN modules and groups, showing their name, ID, and segment ID. The LCN Configuration Panel attempts to derive names from the LCN modules; if a module has no name or is part of a group, a standard name is assigned.
+
+#### Scanning modules
+
+To initiate a scan for LCN modules on the bus, in the top-right corner, select the three dots {% icon "mdi:dots-vertical" %} menu and select **Scan Modules**.
+- **Result**: This process polls each module for its name and serial number. 
+- When all module responses have been received, they will be listed in the device list.
+- Scanning modules may take several seconds. The pop-up dialog will close automatically once the process is complete.
+
+#### Adding devices
+
+If module scanning fails or a module is unavailable on the bus, you can manually add it. Groups can also be created manually.
+
+1. To add a module or group manually, select the **Create Module/Group** button.
+2. Select whether you want to add a module or group and enter the desired `segment id` and module/group `id`. 
+
+    ![Create module/group dialog](/images/integrations/lcn/lcn_create_device.png)
+3. To add the new device, select **Create**.
+
+#### Deleting devices
+
+To delete a single device, select the trash can icon next to it. 
+- **Result**: This will remove the device from the device list and from Home Assistant, including any associated entities.
+
+To delete multiple devices at once, enable selection mode.  Select the desired entries, then, in the top-right corner, select  **Delete Selected**.
+
+### Configuring entities
+
+Entities configured for all devices are listed on the **Entities** tab. 
+
+- To view entities for a specific device (module or group), select the device entry in the **Modules / Groups** tab. This opens the **Entities** tab, filtering entities to show only those for the selected device. 
+- To apply custom filters, enable the filter option.
+
+  ![Create module/group dialog](/images/integrations/lcn/lcn_entities_page.png)
+
+#### Adding entities
+
+1. To create a new entity, select **Create Entity**, which opens the following dialog.
+
+    ![Create entity dialog](/images/integrations/lcn/lcn_create_entity.png)
+2. Select the module or group for which to create the entity from the dropdown menu. 
+    - If a single module or group filter is applied, it will be pre-selected.
```

Comment:
```suggestion
1. To create a new entity, select **Create Entity**.
2. Select the module or group for which to create the entity from the dropdown menu. 
    - If a single module or group filter is applied, it will be pre-selected.

      ![Create entity dialog](/images/integrations/lcn/lcn_create_entity.png)
```